runtime/CSharp/Antlr4.Runtime/Antlr4.Runtime.portable-net45.project.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
```

Comment:
I'm not entirely sure how the implementation relates this dependency with the chosen framework, but .NETStandard 1.0 through 1.6 all use the same declaration here.